Add `.php` stubs for the exposed classes. #14

dom111 opened the pull request.

dom111 committed:
Add `.php` stubs for the exposed classes. #14

kjdev commented:
In the PHP source, it seems to be defined in a file like `xxx.stub.php`.
> ex: https://github.com/php/php-src/blob/master/ext/ffi/ffi.stub.php

I don't see any problem with it as it is, what do you think?

dom111 commented:
Ah, that's a good shout. I didn't look for prior examples. I'll combine to one file later on, won't take long!

dom111 commented:
Updated that to match the format from the other examples. Added some annotations for the mixed returns, moved to the root of the repo and pushed back. Shout if you have any other concerns!
